### PR TITLE
fix: cleaning errors in error log

### PR DESCRIPTION
**Pull request by raxo2222**
Cleaning up some errors

### CODE REVIEW

**Review by JackEPepper — APPROVED** [review]

File: localization/english/CWP/CWP_resolutions_l_english.yml [hunk 1]
```diff
@@ -26,8 +26,20 @@
   concept_un_headquarters_resolution: "UN Headquarters Resolution"
   concept_decolonization_resolution: "Convention on Decolonization"
   concept_apartheid_resolution: "Convention on Apartheid"
-  concept_environmental_modification_resolution: "ENMOD"
+  concept_environmental_modification_resolution: "ENMOD""
```
Comment (DZappala) [inline localization/english/CWP/CWP_resolutions_l_english.yml:29]: extra doublequote?

File: localization/english/CWP/CWP_resolutions_l_english.yml [hunk 2]
```diff
@@ -26,8 +26,20 @@
...
   concept_sustainable_development_resolution: "Sustainable Development"
   concept_unaec_resolution: "UNAEC"
   concept_unga_partition_resolution: "UNGA Partition Resolution"
   concept_resolution_on_korea: "Resolution on Korea"
+
```
Comment (DZappala) [inline localization/english/CWP/CWP_resolutions_l_english.yml:34]: error with this line and below. all yaml lines must be indented to the same level. The indentation of this file skips from 1 space to 0 space. See note below.

File: localization/english/CWP/CWP_resolutions_l_english.yml [hunk 3]
```diff
@@ -26,8 +26,20 @@
...
+  concept_anti_corruption_resolution_desc: "The United Nations Convention Against Corruption (UNCAC) is the only legally binding international anti-corruption multilateral treaty. Negotiated by member states of the United Nations (UN) it has been adopted by the UN General Assembly in October 2003 and entered into force in December 2005. The treaty recognises the importance of both preventive and punitive measures, addresses the cross-border nature of corruption with provisions on international cooperation and on the return of the proceeds of corruption. The UN Office on Drugs and Crime (UNODC) in Vienna serves as Secretariat for the UNCAC. UNCAC's goal is to reduce various types of corruption that can occur across country borders, such as trading in influence and abuse of power, as well as corruption in the private sector, such as embezzlement and money laundering. Another goal of the UNCAC is to strengthen international law enforcement and judicial cooperation between countries by providing effective legal mechanisms for international asset recovery. "
```
Comment (DZappala) [inline localization/english/CWP/CWP_resolutions_l_english.yml:37]: At this line is where the yaml indentation issue begins. eg:

```yaml
l_english:
 key: "content"
 key: "content"
```

File: localization/english/interfaces_l_english.yml [hunk 4]
```diff
@@ -1801,7 +1801,7 @@
  CULTURE_TYPE_DESCRIPTION:0 "Culture, Era [Era.GetNumber]"
  ALREADY_ACQUIRED_BUTTON:0 "See Known Technologies"
 
- TECHNOLOGY_ACQUIRED_POPUP_HEADER:1 "#header [PlayerMessageItem.GetScopeTechnology.GetName] Unlocked#!"
+ TECHNOLOGY_AQUIRED_POPUP_HEADER:1 "#header [PlayerMessageItem.GetScopeTechnology.GetName] Unlocked#!"
```
Comment (DZappala) [inline localization/english/interfaces_l_english.yml:1804]: The related section in `.gui` file should be updated to reflect the correct spelling: `Acquired`

**Review by DZappala — CHANGES_REQUESTED** [review]

File: common/country_definitions/00_countries.txt [hunk 5]
```diff
@@ -6425,7 +6425,7 @@ BOZ = {
 	country_type = colonial
 	tier = principality
 	cultures = {
-		english
+		british
```
[the remark this answers is not in the document]
Comment (raxo2222) [reply common/country_definitions/00_countries.txt:6428]: English culture doesn't exist - it was causing error in error log

File: localization/english/CWP/CWP_resolutions_l_english.yml (hunk 1 above)
Comment (raxo2222) [reply localization/english/CWP/CWP_resolutions_l_english.yml:29]: Accidental key press I guess

File: localization/english/interfaces_l_english.yml (hunk 4 above)
Comment (raxo2222) [reply localization/english/interfaces_l_english.yml:1804]: Its in vanilla - gui file isn't present in mod

File: common/country_definitions/00_countries.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (raxo2222) [reply common/country_definitions/00_countries.txt:6414]: Renamed British to English culture

File: common/flag_definitions/BU/00_flag_definitions.txt
[the remark this answers is not in the document]
Comment (raxo2222) [reply common/flag_definitions/BU/00_flag_definitions.txt:1]: Restored and added comment on top of it so we can know its purpose

File: localization/english/CWP/CWP_defcon_l_english.yml
[the remark this answers is not in the document]
Comment (raxo2222) [reply localization/english/CWP/CWP_defcon_l_english.yml:1]: done

File: localization/english/CWP/CWP_resolutions_l_english.yml (hunk 3 above)
Comment (raxo2222) [reply localization/english/CWP/CWP_resolutions_l_english.yml:37]: Cleaned it up

File: localization/english/interfaces_l_english.yml (hunk 4 above)
Comment (raxo2222) [reply localization/english/interfaces_l_english.yml]: Added vanilla file as to fix typo anyway

File: common/pop_needs/00_pop_needs.txt
Comment (JackEPepper) [inline common/pop_needs/00_pop_needs.txt:1]: Why was this changed?

Comment (raxo2222) [reply common/pop_needs/00_pop_needs.txt:1]: Consider situation:
You are playing very backwards nation, that doesn't have railways yet, so no transportation.
Or you just don't have infrastructure yet.

You also blocked services production by enabling housing and professional services.
This means no goods on popneed_free_movement and popneed_communication  are present in market.
Game will use default good, as buy orders is a must.
I changed fallback good to services.

Comment (JackEPepper) [reply common/pop_needs/00_pop_needs.txt:1]: Okay, fair enough

**Review by DZappala — APPROVED** [review]
